<!--
  2 failure:  :microscope: Ther..., :disappointed_rel...
  1 warning:  :mag: Can't find ...
  1 messages
  
  DangerID: danger-id-Danger;
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  :microscope: There are library changes, but no changes to the unit tests. That's OK as long as you're refactoring existing code, but will require an admin to merge this PR. Please see <a href="https://github.com/janvennemann/titanium_mobile/blob/TIMOB-27175-8_1_X/README.md#unit-tests">README.md#unit-tests</a> for docs on unit testing.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  :disappointed_relieved: `npm test` failed. See below for details.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  :mag: Can't find junit reports at `./junit.*.xml`, skipping generating JUnit Report.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  ```
> titanium-mobile@8.1.0 test /Users/build/jenkins/workspace/ium-sdk_titanium_mobile_PR-10993
> npm run ios-sanity-check && grunt


> titanium-mobile@8.1.0 ios-sanity-check /Users/build/jenkins/workspace/ium-sdk_titanium_mobile_PR-10993
> ./build/scons check-ios-toplevel

Running "appcJs:src:lintOnly" (appcJs) task

Running "eslint:src" (eslint) task

/Users/build/jenkins/workspace/ium-sdk_titanium_mobile_PR-10993/cli/lib/tasks/process-js-task.js
  150:12  warning  Avoid calling back inside of a promise  promise/no-callback-in-promise
  157:5   warning  Avoid calling back inside of a promise  promise/no-callback-in-promise

/Users/build/jenkins/workspace/ium-sdk_titanium_mobile_PR-10993/iphone/cli/commands/_build.js
  5884:15  warning  Avoid calling back inside of a promise                                  promise/no-callback-in-promise
  6411:79  warning  'out' is defined but never used. Allowed unused args must match /^_.+/  no-unused-vars

/Users/build/jenkins/workspace/ium-sdk_titanium_mobile_PR-10993/iphone/cli/hooks/frameworks.js
   60:5   warning  Expected catch() or return                  promise/catch-or-return
   60:34  warning  Avoid calling back inside of a promise      promise/no-callback-in-promise
  425:5   warning  Each then() should return a value or throw  promise/always-return

/Users/build/jenkins/workspace/ium-sdk_titanium_mobile_PR-10993/tests/Resources/ti.network.httpclient.addontest.js
  26:26  warning  'e' is defined but never used. Allowed unused args must match /^_.+/  no-unused-vars

✖ 8 problems (0 errors, 8 warnings)


Running "checkFormat:ios" (checkFormat) task
Fatal error: Error: Formatting incorrect on "iphone/Classes/TiUtils+Addons.m", proposed changes: <?xml version='1.0'?>
<replacements xml:space='preserve' incomplete_format='false'>
<replacement offset='656' length='6'>&#10;&#10;  </replacement>
</replacements>

npm ERR! Test failed.  See above for more details.

```
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a152faccdbefaac6ab3b5281436c12f767f2fb5e
</p>

--- a/iphone/Classes/TiUtils+Addons.m
+++ b/iphone/Classes/TiUtils+Addons.m
@@ -20,7 +20,7 @@ static NSString *kAppUUIDString = @"com.appcelerator.uuid";
     [defaults setObject:uid forKey:kAppUUIDString];
     [defaults synchronize];
   }
-  
+
   return uid;
 }
 

